features/Kiesrecht.feature

```diff
@@ -1,38 +1,101 @@
 ﻿# language: nl
 
 Functionaliteit: Kiesrecht
+	De API toont of een persoon kiesrecht heeft.
 
-Scenario: de ingeschreven persoon is uitgesloten van europees kiesrecht
+	Het kiesrecht wordt afgeleid uit gegevens in de registratie. Voor Europees kiesrecht kan in de registratie zijn vastgelegd of een persoon een oproep krijgt dan wel is uitgesloten. Voor (niet-Europees) kiesrecht kan in de registratie zijn vastgelegd dat een persoon is uitgesloten.
+	Er worden geen aannames gedaan over (binnenlands of Europees) kiesrecht wanneer de registratie daar geen gegevens over heeft vastgelegd. In de API wordt kiesrecht en Europees kiesrecht alleen gevuld met gegevens die direct en zeker zijn af te leiden van wat in de registratie is vastgelegd.
+
+	De actuele situatie wordt getoond. Dus wanneer een uitsluiting in het verleden ligt, wordt die uitsluiting niet opgenomen in de API.
+
+	Opname van onbekende datums gebeurt op dezelfde manier als elders in de API.
+
+Scenario: de ingeschreven persoon is uitgesloten van Europees kiesrecht
 	Gegeven op de PL van een ingeschreven persoon is categorie 13, element 31.10 gelijk aan 1 (= persoon is uitgesloten)
 	Als de ingeschreven persoon wordt geraadpleegd
 	Dan is europeesKiesrecht gelijk aan false
 
-Scenario: de ingeschreven persoon is niet uitgesloten van europees kiesrecht (1)
+Scenario: de ingeschreven persoon ontvangt oproep voor Europees kiesrecht
 	Gegeven op de PL van een ingeschreven persoon is categorie 13, element 31.10 gelijk aan 2 (= persoon ontvangt oproep)
 	Als de ingeschreven persoon wordt geraadpleegd
 	Dan is europeesKiesrecht gelijk aan true
 
-Scenario: de ingeschreven persoon is niet uitgesloten van europees kiesrecht (2)
+Scenario: de ingeschreven persoon is niet uitgesloten van Europees kiesrecht

```

Comment:
Moet de titel van de scenario niet zijn a la: uitsluiting van Europees kiesrecht is onbekend? Dit omdat 31.10 leeg is. europeesKiesrecht attribuut is ook niet aanwezig.

Reply:
dank voor de tip, ik pas het aan